cnames_active.js

```diff
@@ -2103,6 +2103,7 @@ var cnames_active = {
   "onemsg": "7f4243c7-d171-416a-ad3d-19ba04fe36f5.id.repl.co", // noCF
   "ongaku": "anshuman-verma.github.io/ongaku",
   "onii": "oniijs.github.io/onii",
+  "oo": "mhsarmadi.github.io/oo"
```

Comment:
```suggestion
  "oo": "mhsarmadi.github.io/oo",
```
Your entry needs a comma at the end of the line